options/parse_configfile.c

```diff
@@ -163,16 +163,19 @@ int m_config_parse_config_file(m_config_t *config, struct mpv_global *global,
 
     MP_VERBOSE(config, "Reading config file %s\n", conffile);
 
+    int r = 0;
+
     struct stream *s = stream_create(conffile, STREAM_READ | STREAM_ORIGIN_DIRECT,
                                      NULL, global);
     if (!s)
-        return 0;
+        goto done;
+    stream_skip_bom(s);
```

Comment:
fyi, m_config_parse already skips bom, so this changes nothing, I don't oppose it for consistency sake.

Reply:
Huh yeah. I'll just drop the commit then.